.github/workflows/test.yml

```diff
@@ -25,7 +38,13 @@ jobs:
         python -m pip install --upgrade pip
         pip install tox
     - name: Execute tox
-      env:
-        CI: "true"
```

Comment:
GitHub Actions documentation guarantees that `CI == "true"`

See https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables